ui/src/CurrentUser.ts

```diff
@@ -124,4 +126,23 @@ export class CurrentUser {
             .post(config.get('url') + 'current/user/password', {pass})
             .then(() => this.snack('Password changed'));
     };
+
+    public tryReconnect = (quiet: boolean = false) => {
```

Comment:
The boolean type is not needed. tslint complains about that https://travis-ci.org/gotify/server/builds/597248630#L1798

Reply:
Yeah, I've made a mistake about that.